documents/Libraries/sxpbrlib/sxpbrlib_defs.mtlx

```diff
@@ -240,54 +240,54 @@
   -->
   <nodedef name="ND_standard_surface_surfaceshader" node="standard_surface" type="surfaceshader" nodegroup="pbr"
            doc="A surface uber shader based on the Arnold standard surface shader">
-      <input name="base" type="float" value="0.8" />
-      <input name="base_color" type="color3" value="1, 1, 1" />
-      <input name="diffuse_roughness" type="float" value="0" />
-      <input name="specular" type="float" value="1" />
-      <input name="specular_color" type="color3" value="1, 1, 1" />
-      <input name="specular_roughness" type="float" value="0.1" />
-      <input name="specular_IOR" type="float" value="1.52" />
-      <input name="specular_anisotropy" type="float" value="0" />
-      <input name="specular_rotation" type="float" value="0" />
-      <input name="metalness" type="float" value="0" />
-      <input name="transmission" type="float" value="0" />
-      <input name="transmission_color" type="color3" value="1, 1, 1" />
-      <input name="transmission_depth" type="float" value="0" />
-      <input name="transmission_scatter" type="color3" value="0, 0, 0" />
-      <input name="transmission_scatter_anisotropy" type="float" value="0" />
-      <input name="transmission_dispersion" type="float" value="0" />
-      <input name="transmission_extra_roughness" type="float" value="0" />
-      <input name="subsurface" type="float" value="0" />
-      <input name="subsurface_color" type="color3" value="1, 1, 1" />
-      <input name="subsurface_radius" type="color3" value="1, 1, 1" />
-      <input name="subsurface_scale" type="float" value="1" />
-      <input name="thin_walled" type="boolean" value="false" />
-      <input name="normal" type="vector3">
+      <input name="base" type="float" value="0.8" uiname="Base" uifolder="Base"/>
+      <input name="base_color" type="color3" value="1, 1, 1" uiname="Base Color"  uifolder="Base"/>
+      <input name="diffuse_roughness" type="float" value="0" uiname="Diffuse Roughness"  uifolder="Diffuse"/>
```

Comment:
There's a lot of inputs so adding a uifolder for testing ui grouping which can try in the viewer.